send_message.py

```diff
@@ -33,14 +29,17 @@ def send_message(camera, timestamp, feed_url, img_filename):
 
     msg.attach(image)
 
-    # recipient = phone_number + CARRIERS[carrier]
-    auth = (EMAIL, PASSWORD)
+    email = vault.get_value("sms", 0)
+    password = vault.get_value("sms", 1)
+    recipients = [vault.get_value(person) + env.CARRIERS[carrier] for person in env.RECIPIENTS.keys()] 
```

Comment:
We should get these values through the api rather than directly :D

Reply:
We are now retrieving all env variables through security API